docs/mkdocs.base.yml

```diff
@@ -1,6 +1,6 @@
 site_name: Quantum Ready Architecture for Drug Discovery
 site_url: https://awslabs.github.io/quantum-ready-solution-for-drug-discovery/
-repo_url: https://github.com/awslabs/quantum-ready-solution-for-drug-discovery
+repo_url: https://github.com/awslabs/quantum-ready-solution-for-drug-discovery/
 
 theme:
   name: material
```

Comment:
Why does 'https://github.com/awslabs/quantum-ready-solution-for-drug-discovery' still work?

Reply:
it's best practise.